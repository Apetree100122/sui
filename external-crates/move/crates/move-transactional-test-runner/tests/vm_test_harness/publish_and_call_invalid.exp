processed 1 task

task 0, lines 1-10:
//# publish-and-call --call 0x42::m::a --call 0x42::m::d
Error: Function execution failed with VMError: {
    major_status: UNKNOWN_INVARIANT_VIOLATION_ERROR,
    sub_status: None,
    location: undefined,
    indices: [],
    offsets: [],
}
