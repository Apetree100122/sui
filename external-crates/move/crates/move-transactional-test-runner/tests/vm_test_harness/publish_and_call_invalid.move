//# publish-and-call --call 0x42::m::a --call 0x42::m::d
module 0x42::m {
    fun a(): u64 {
        0
    }

    fun b(): u64 {
        0
    }
}
