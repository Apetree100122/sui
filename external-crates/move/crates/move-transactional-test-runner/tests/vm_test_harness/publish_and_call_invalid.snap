---
source: crates/move-transactional-test-runner/src/framework.rs
---
processed 1 task

task 0, lines 1-10:
//# publish-and-call --call 0x42::m::a --call 0x42::m::d
Error: Function execution failed with VMError: {
    major_status: VTABLE_KEY_LOOKUP_ERROR,
    sub_status: None,
    location: undefined,
    indices: [],
    offsets: [],
    message: "Could not find function 0000000000000000000000000000000000000000000000000000000000000042::m::d",
}
